Platform/ElkhartlakeBoardPkg/AcpiTables/AcpiTables.inf

```diff
@@ -0,0 +1,62 @@
+## @file
+# Component description file for PlatformAcpiTable module.
+#
+#  Copyright (c) 1999 - 2020, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = AcpiTables
+  FILE_GUID                      = 7E374E25-8E01-4FEE-87F2-390C23C606CD
+  MODULE_TYPE                    = USER_DEFINED
+  VERSION_STRING                 = 1.0
+  EDK_RELEASE_VERSION            = 0x00020000
+  EFI_SPECIFICATION_VERSION      = 0x00020000
+
+[sources.common]
+  AcpiTablePlatform.h
+  Madt/Madt.h
+  Dsdt/PlatformBoardId.h
+  Dmar/Dmar.h
+  Dsdt/GpioAcpiDefinesVer3.h
+  Dsdt/PchSerialIoAcpiDefinesEhl.h
+  Dsdt/Dsdt.asl
+  Facs/Facs.aslc
+  Fadt/Fadt.aslc
+  Fwst/Fwst.aslc
+  Madt/Madt30.aslc
+  Mcfg/Mcfg.aslc
+  Hpet/Hpet.aslc
+  Platform/CommonBoardPkg/AcpiTables/Fpdt/Fpdt.aslc
+  Ssdt/SaSsdt.asl
+  SsdtRtd3/EhlCrbRtd3.asl
+  Dmar/Dmar.aslc
+  CpuSsdt/Cpu0Cst.asl
+  CpuSsdt/Cpu0Ist.asl
+  CpuSsdt/Cpu0Psd.asl
+  #CpuSsdt/Cpu0Tst.asl
+  CpuSsdt/ApCst.asl
+  CpuSsdt/ApIst.asl
+  CpuSsdt/ApPsd.asl
+  #CpuSsdt/ApTst.asl
+  CpuSsdt/CpuSsdt.asl
+  #CpuSsdt/Itbm.asl
+  #CpuSsdt/ItbmSci.asl

```

Comment:
Remove these commented Lines.
And also remove these unused ASL files

Reply:
Done